Failure building with --features=race

```console
bazel build --features=race //cmd/bla
INFO: Analysed 4 targets (292 packages loaded).
INFO: Found 4 targets...
ERROR: /private/var/tmp/_bazel_mmazurskiy/b34360f02dd290fea1de7cb4ffb49b60/external/io_bazel_rules_go/go/tools/builders/BUILD.bazel:79:1: error executing shell command: 'export GOROOT=$(pwd)/bazel-out/host/bin/external/go_stdlib_darwin_amd64_cgo_race && bazel-out/host/bin/external/go_stdlib_darwin_amd64_cgo_race/bin/go tool compile -o bazel-out/host/bin/external/io...' failed (Exit 1)
external/io_bazel_rules_go/go/tools/builders/env.go:18:2: can't find import: "flag"
INFO: Elapsed time: 1.358s, Critical Path: 0.37s
FAILED: Build did NOT complete successfully
```
If I build the same package without `--features=race` - it works.

OS: macOS 10.12.6
Bazel: 0.8.1 
Rules_go: latest as of today dfcdede243e986a0dd1e4c3e2f1005051ab5b15e

p.s. another thing I have noticed: `race` parameter to `go_test`/`go_binary` is not documented anywhere. Or I cannot find it? Is using it for binaries is the recommended way (like for `go_test`)?

--- a/go/private/go_toolchain.bzl
+++ b/go/private/go_toolchain.bzl
@@ -45,6 +45,7 @@ def _go_toolchain_impl(ctx):
   return [platform_common.ToolchainInfo(
       name = ctx.label.name,
       cross_compile = ctx.attr.cross_compile,
+      bootstrap = ctx.attr.bootstrap,
       default_goos = ctx.attr.goos,
       default_goarch = ctx.attr.goarch,
       stdlib = struct(

--- a/go/private/mode.bzl
+++ b/go/private/mode.bzl
@@ -57,6 +57,7 @@ def get_mode(ctx, toolchain_flags):
 
   # We always have to use the pure stdlib in cross compilation mode
   force_pure = "on" if go_toolchain.cross_compile else "auto"
+  force_race = "off" if go_toolchain.bootstrap else "auto"
 
   #TODO: allow link mode selection
   static = _ternary(
@@ -65,6 +66,7 @@ def get_mode(ctx, toolchain_flags):
   )
   race = _ternary(
       getattr(ctx.attr, "race", None),
+      force_race,
       "race" in ctx.features,
   )
   msan = _ternary(
@@ -76,6 +78,9 @@ def get_mode(ctx, toolchain_flags):
       force_pure,
       "pure" in ctx.features,
   )
+  if race and pure:
+    # You are not allowed to compile in race mode with pure enabled
+    race = False
   debug = ctx.var["COMPILATION_MODE"] == "debug"
   strip_mode = "sometimes"
   if toolchain_flags:

